frontend/src/app/inbox/ChatPreview.tsx

```diff
@@ -25,6 +28,8 @@ export default function ChatPreview({ match, users, community }) {
     const chatTitle = getChatTitle(matchedUsers)
     const userDiscs = matchedUsers.map((u) => <UserDisc key={u.uid} user={u} />)
     const userClass = matchedUsers.length === 1 ? 'Single' : 'Group'
+    const authUser = useCurrentAuthUser()
+    const currentChat: ChatData = chatData
```

Comment:
This is just a variable rename, if you want to show the type of the arguments you can do this:

```tsx
interface ChatPreviewProps {
  match: ChatData
  users: { [userId: string]: UserData }
  community: CommunityData
}

export default function ChatPreview({ match, users, community, chatData }: ChatPreviewProps) { ... }
```